### PR TITLE
Reference field nameGets batched multiEdit

### PR DESCRIPTION
**Pull request by kebeclibre**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kebeclibre** [comment]
robodoo rebase-ff

**Comment by kebeclibre** [comment]
@aab-odoo I know why it is red. should not we backport 652414ca4e42c0f0ece0b70e88a1a74c014b0dcd ?
cc @Arcasias

File: addons/web/static/tests/views/list_tests.js [hunk 1]
```diff
@@ -4660,7 +4660,7 @@ QUnit.module('Views', {
                     if (args.model === 'bar') {
                         assert.deepEqual(args.args[0], [1, 2 ,3]);
                     }
-                    if (args.model === "res.currency") {
+                    if (args.model === "res_currency") {
```
Comment (aab-odoo) [inline addons/web/static/tests/views/list_tests.js:4663]: lol bien vu

**Review by aab-odoo — COMMENTED** [review]
multi edit in saas-12.3?? This feature appeared in 13.0 :-)

That probably explains why it's red ^^

About the backport, that commit fixes a different issue, isn't it? That's easy to check, with the tests. In any cases, it could (should?) be backported to 13.0.

**Comment by kebeclibre** [comment]
@fw-bot ignore

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**